api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -252,6 +255,7 @@ public class ApiConstants {
     public static final String HYPERVISOR = "hypervisor";
     public static final String INLINE = "inline";
     public static final String INSTANCE = "instance";
+    public static final String INSTANCE_ID = "instance_id";
```

Comment:
Hi @bernardodemarco there is a mismatch between this parameter value and the autocompletion cmk PR name (`instance_id` vs `instanceid`). For simplicity, can this value be changed to `instanceid`?